swift/ql/src/diagnostics/SuccessfullyExtractedLines.ql

```diff
@@ -0,0 +1,15 @@
+/**
+ * @name Successfully extracted lines
+ * @description Count all lines in source code in which something was extracted.
+ * @kind metric
+ * @id swift/diagnostics/successfully-extracted-lines
+ * @tags summary
```

Comment:
Please confirm that ` *       lines-of-code` has been omitted here to avoid code-scanning from picking this up for the time being.

Reply:
yes, this query is different than `lines-of-code` queries from other languages, as
* it will not count lines for which nothing was extracted because of any kind of problem
* it does count comment lines
I think this might be confusing if it were presented as the user-facing lines-of-code, but maybe this is up for debate? For the time being I'd keep it separate

Reply:
For context: I've looked into how the C/C++ extractor extracts the data used by `lines-of-code`, and on each source file it just bluntly opens the file and counts the lines, matching empty and comment lines to fill in the different values of the [numlines table].(https://github.com/github/codeql/blob/33b4d2d6531d62f6716d4ea05dc57777795568e7/cpp/ql/lib/semmlecode.cpp.dbscheme#L256-L262)